File: Beginner/Fuel_Spent_1017/main.go
package main

import (
	"fmt"
)

const fuelSpent = 12

func main() {
	var spentTime float64
	fmt.Scan(&spentTime)
	var averageSpeed float64
	fmt.Scan(&averageSpeed)
	distance := averageSpeed * spentTime / fuelSpent
	fmt.Printf("%.3f\n", distance)
}

File: Beginner/Fuel_Spent_1017/Main.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        final int spentFuel = 12;
        Scanner sc = new Scanner(System.in);
        double spentTime = sc.nextInt();
        double averageSpeed = sc.nextInt();

        double distance = spentTime * averageSpeed / spentFuel;
        System.out.printf("%.3f\n", distance);
        sc.close();
    }
}